packages/lesswrong/components/ea-forum/EASequencesHome.tsx

```diff
@@ -5,7 +5,8 @@ import { AnalyticsContext } from "../../lib/analyticsEvents";
 const styles = (theme: ThemeType): JssStyles => ({
   description: {
     marginTop: theme.spacing.unit,
-    marginBottom: theme.spacing.unit*4,
+    marginBottom: `${theme.spacing.unit * 2}px !important`,
```

Comment:
Who are we arguing with? Can we win with the `&&` trick? (I forget the precise details of the trick.)